cmd/pulumi.go

```diff
@@ -232,6 +235,15 @@ func getCLIVersionInfo() (semver.Version, semver.Version, error) {
 		return semver.Version{}, semver.Version{}, err
 	}
 
+	brewLatest, isBrew, err := getLatestBrewFormulaVersion()
```

Comment:
Do we want to avoid calling our service if `isBrew`? Or is there a reason to always do it (e.g. for version usage telemetry)?

Reply:
The latter.